### PR TITLE
Gci 1302 consume chd item ordered topic

### PR DESCRIPTION
**Pull request by nabeelamirch**
Consume chd item ordered topic.

resolves: GCI-1302

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -76,6 +92,34 @@
...
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
+            <artifactId>powermock-module-junit4</artifactId>
+            <version>${powermock.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
```
Comment (caismith-ch) [inline pom.xml:102]: Doesn't look like this dependency so can be removed but we should try not to use powermock as in the past we have had to strip it from our services https://companieshouse.atlassian.net/wiki/spaces/DEV/pages/936345626/Removing+Powermock

Comment (nabeelamirch) [reply pom.xml]: I can't remember why I added it in. I've removed it now.

File: src/test/java/uk/gov/companieshouse/chdorderconsumer/kafka/ItemOrderedKafkaConsumerTest.java [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class ItemOrderedKafkaConsumerTest {
+    private static final String ORDER_RECEIVED_URI = "/order/ORDER-12345";
```
Comment (VaughanJackson123) [inline src/test/java/uk/gov/companieshouse/chdorderconsumer/kafka/ItemOrderedKafkaConsumerTest.java:36]: `ORDER_RECEIVED_URI` - we might as well make this a realistic URI here for the avoidance of doubt?

Comment (VaughanJackson123) [reply src/test/java/uk/gov/companieshouse/chdorderconsumer/kafka/ItemOrderedKafkaConsumerTest.java]: Sorry, should be clearer about this. The actual order URIs begin `/orders/..`?

Comment (nabeelamirch) [reply src/test/java/uk/gov/companieshouse/chdorderconsumer/kafka/ItemOrderedKafkaConsumerTest.java]: you're right totally missed that.

**Review by VaughanJackson123 — APPROVED** [review]

**Pull request merged**